common/test/acceptance/tests/studio/test_studio_tabs.py

```diff
@@ -99,11 +99,87 @@ def test_user_can_reorder_static_tabs(self):
         self.pages_page.set_field_val("Display Name", "First")
         self.pages_page.save()
         self.pages_page.add_static_page()
-        self.pages_page.drag_and_drop_first_static_page_to_last()
+        self.pages_page.drag_and_drop()
         self.pages_page.refresh_and_wait_for_load()
         static_tab_titles = self.pages_page.static_tab_titles
         self.assertEqual(
             static_tab_titles,
             ['Empty', 'First'],
             'Order should be:["Empty", "First] but getting {} from the page'.format(static_tab_titles)
         )
+
+    def test_user_can_reorder_builtin_tabs(self):
+        """
+        Scenario: Users can reorder built-in pages
+            Given I have opened the pages page in a new course
+                Then the built-in pages are in the default order
+            When I drag the first page to the last
+                Then the built-in pages are switched
+            And I reload the page
+                Then the built-in pages are switched
+        """
+        default_order = ['Home', 'Course', 'Discussion', 'Wiki', 'Progress']
+        new_order = ['Home', 'Course', 'Wiki', 'Progress', 'Discussion']
+        default_tabs_order = self.pages_page.default_tab_titles
+        self.assertEqual(
+            default_tabs_order,
+            default_order,
+            'Tabs are not in the default order'
+        )
+        self.pages_page.drag_and_drop(default_tab=True)
+        default_tabs_order = self.pages_page.default_tab_titles
+        self.assertEqual(
+            default_tabs_order,
+            new_order,
+            'Tabs are not in the new order'
+        )
+        self.pages_page.refresh_and_wait_for_load()
+        self.assertEqual(
+            default_tabs_order,
+            new_order,
+            'Tabs are not in the new order'
+        )
+
+    def test_users_can_toggle_visibility(self):
+        """
+        Scenario: Users can toggle visibility on hideable pages
+        Given I have opened the pages page in a new course
+            Then I should see the "wiki" page as "visible"
+        When I toggle the visibility of the "wiki" page
+            Then I should see the "wiki" page as "hidden"
+        And I reload the page
+            Then I should see the "wiki" page as "hidden"
+        When I toggle the visibility of the "wiki" page
+            Then I should see the "wiki" page as "visible"
+        And I reload the page
+            Then I should see the "wiki" page as "visible"
+        """
+        tab = 'wiki'
+        self.assertFalse(self.pages_page.check_tab_visibility_status(tab))
```

Comment:
Either the comments have a different order or the test is faulty, the comments start with 
Given I have opened the pages page in a new course
            Then I should see the "wiki" page as "visible"

but the test starts with
tab = 'wiki'
self.assertFalse(self.pages_page.check_tab_visibility_status(tab))

so the comments assume that default selected tab should be wiki but test assume it isn't

Reply:
yes, I have updated the method name. The test is doing fine, the confusion was with the method name